### ISSUE TITLE
Chrome suppport

### ISSUE THREAD

**Issue by almadimate** [krisztianb/seekingalpha-paywall-remover#3]
Hi Krisztián,
Do you plan to make it available for Google Chrome? Version 1.5 worked great with Chrome if I loaded manually but the latest does not. Best Regards,
Mate

**Comment by krisztianb** [comment]
Hi Mate. That is strange because there were no meaningful changes in version 1.6. Have you tried reinstalling and/or deleting your cookies in the browser?

It shouldn't be too difficult to create a Chrome version however I have no plans of doing so currently.

**Comment by almadimate** [comment]
Yes, I tried removing cookies, etc but the same behavior with version 1.6: Reading an article the whole site is frozen, scrolling is not working, top search bar and left side of the page remain active only. Nevertheless if you don't plan to make it available for Chrome then it's just my problem. :) I prefer Chrome because I just right click on the page and it translates to the preferred language.

![image](https://github.com/user-attachments/assets/beaade52-2ea5-4463-ad38-6ff9b673a2d3)

**Comment by almadimate** [comment]
Looks like this change solved my problem in Chrome, thanks ChatGPT :D

const adSelector = "main + div, main > div";

**Comment by krisztianb** [comment]
Ok, I don't understand how that fixes anything. But if it works for you, great. :smile:
